packages/api-client/src/odsql/index.ts

```diff
@@ -134,24 +135,14 @@ export class Query {
 function root(source: string) {
     return () => ({
         itself: () => new Query(`${source}/`),
-        /**
-         * @deprecated Use query() instead
-         */
-        aggregates: () => new Query(`${source}/aggregates/`),
         facets: () => new Query(`${source}/facets/`),
         datasets: () => new Query(`${source}/datasets/`),
-        query: () => new Query(`${source}/query/`),
-        exports: (format: string) => new Query(`${source}/exports/${format}/`),
+        exports: (format: ExportCatalogFormat) => new Query(`${source}/exports/${format}/`),
```

Comment:
This could be outside the scope of this PR and is kind of eye-candy, but it would be a breaking change so it may be a good time. For datasets, we have:
- `.datasets()` matching `/datasets/`
- `.dataset('xxx')` matching `/datasets/xxx/`

Maybe for exports it would make more sense and be more intuitive to have `.export('geojson')` (singular) matching `.../exports/geojson/`. What do you think?

Reply:
Hmm I think you're right, this is the best moment to do this.